disruption_generator/cli.py

```diff
@@ -39,6 +39,12 @@ def main(experiments_path):
 
 
 async def execute(experiments_path):
+    """Find, parse and execute experiments.
+
+    :param experiments_path: Path to directory with definitions of experiments.
+    :return: 0 on success, else 1.
+    :rtype: int
+    """
```

Comment:
This is just an example of how docstring should look like for autodoc make proper use of it.